### PR TITLE
Language Vote Implimenation

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Impliments the changes done in the voting channel.

Esperanto -> Lingua Romana
German -> Eurolang (tiebreaker at the time of calling the vote)
Techno-Russian -> Interslavic
Serbian -> Illyrian

## Changelog
:cl:
tweak: Renames languages and changes language call-procs to their new names.
tweak: Removes the 'ancient' from Latin. No one speaks Roman 'ancient' Latin, we barely even know today what it sounds like properly.
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
I will still call it Interromanico and nothing will stop me

File: code/modules/mob/language/ancient_latin.dm [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /datum/language/ancientlatin
 	name = LANGUAGE_LATIN
-	desc = "An ancient language once used by most of europe but now mainly used by followers of the Church of Absolute."
+	desc = "An ancient language once used by most of europe and accidemics but now mainly used by followers of Christian branches and the Church of Absolute."
```
Comment (elgeonmb) [inline code/modules/mob/language/ancient_latin.dm:3]: *academics

File: code/modules/mob/language/illyrian.dm [hunk 2]
```diff
@@ -1,14 +1,14 @@
 
-/datum/language/serbian
-	name = LANGUAGE_SERBIAN
-	desc = "Language popular among Serbian mercenaries, pirates, and criminals."
-	colour = "serbian"
+/datum/language/illyrian
+	name = LANGUAGE_ILLYRIAN
+	desc = "A language derived from Earth's Balakan regions. Commonly spoken as a linguia franka among urban background mercenaries, pirates, and within criminal trades."
```
Comment (elgeonmb) [inline code/modules/mob/language/illyrian.dm:4]: *balkan
*lingua franca

File: code/modules/mob/language/outsider.dm [hunk 3]
```diff
@@ -111,7 +111,7 @@
 //Kriosan racial language. Lore: German creolization due to Sol-Gov occupation; therefor intelligibility with German.
 /datum/language/kriosan
 	name = LANGUAGE_KRIOSAN
-	desc = "The main language of Krios, Kriosan Creole. Mostly consiting of quill-hound noises, posturing and many German borrow words."
+	desc = "The main language of Krios, Kriosan Creole. Mostly consiting of quill-hound noises, posturing and many Eurolang borrow words."
```
Comment (elgeonmb) [inline code/modules/mob/language/outsider.dm:114]: theyre called loan words not borrow words??? might as well fix that while you're in here

File: code/modules/mob/language/station.dm [hunk 4]
```diff
@@ -92,12 +92,12 @@
 
 /datum/language/russian
 	name = LANGUAGE_CYRILLIC
-	desc = "Ancient language of Russian colonists, rusted with time and bastardized with technical terms in everyday use."
-	colour = "russian"
+	desc = "A language derived from Earth's Eastern European and Central Asian regions. Commonly spoken in forntier colonies and finding its home on many agricultural worlds."
```
Comment (elgeonmb) [inline code/modules/mob/language/station.dm:95]: probably want that last sentence to be

"Commonly spoken in frontier colonies, it finds its home on many agricultural worlds."

**Comment by Stutternov** [comment]
Going to make a small change today and it should be ready after.

**Pull request merged**